code/game/objects/structures/watercloset.dm

```diff
@@ -120,8 +116,14 @@
 		w_items += secret.w_class
 		contents += secret
 
+/obj/structure/toilet/secret/prison //We care about you Skyrats!
```

Comment:
remove the comment

Reply:
> 
> 
> remove the comment
@kevinz000 What exactly is wrong with adding a comment that nods to one of your downstreams?

Reply:
comments are for clarifying what code is doing or why it's doing it and this does neither, it just sorta says we care about skyrat, which, yeah, we totally should, but without the context of this PR that comment makes zero sense